packs/clicrud/config.schema.yaml

```diff
@@ -11,14 +11,17 @@
     secret: true
     required: true
   enable:
-    description: "Authentication method."
+    description: "Enable password."
```

Comment:
Thanks for fixing that, I misunderstood what it is for (I did quickly look at clicrud repo, but it still wasn't immediately obvious :D)